src/Microsoft.Dnx.TestHost/Project.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright(c) .NET Foundation.All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Microsoft.Dnx.TestHost
+{
+    public class Project
+    {
+        private const string ProjectFileName = "project.json";
+
+        public string Name { get; set; }
+
+        public IDictionary<string, string> Commands { get; }
+            = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+
+        public static bool TryGetProject(string inputPath, out Project project)
+        {
+            project = null;
```

Comment:
Don't init this until you have to, then the compiler will force you to init it on every exit point. This makes it harder to introduce bugs